extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -487,6 +493,12 @@ export class AmpA4A extends AMP.BaseElement {
               creativeParts.creative) {
             this.creativeBody_ = creativeParts.creative;
           }
+          if (creativeParts && creativeParts.size
```

Comment:
I think that while it may seem messier, it's better to do it that way then check the same thing over and over.